src/QueryM.purs

```diff
@@ -196,6 +196,15 @@ data FeeEstimateError
   = FeeEstimateHttpError Affjax.Error
   | FeeEstimateDecodeJsonError Json.JsonDecodeError
 
+-- No Show instance of Affjax.Error
+instance Show FeeEstimateError where
```

Comment:
Let's use Show to output strings that are as close to code used to construct the value as possible.

Let's add a new PrettyPrint typeclass for human-readable strings, if this use is common.

Reply:
I see, we don't have a valid Show instance for Affjax Error... Can't be helped then.